static/js/main.js

```diff
@@ -1,3 +1,6 @@
+let previousX;
+let previousY;
```

Comment:
Just leaving this as a note for next iteration. I bet we could call jquery's click functions to kickstart that change happening and leave `previousX` and `previousY` in that function, but this is fine for now